server/models/Profile.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const profileSchema = new mongoose.Schema({
+  firstname: {
```

Comment:
camelCase should be used in the properties. `firstName`, applicable throughout.

Reply:
fixed that in other branches when i noticed it as well